data_validation/partition_builder.py

```diff
@@ -40,17 +41,38 @@ def _get_arg_config_dir(self) -> str:
 
         return self.args.config_dir
 
-    def _get_yaml_from_config(self, config_manager: ConfigManager) -> Dict:
-        """Return dict objects formatted for yaml validations.
+    def _add_filters_get_yaml(
+        self,
+        config_manager: ConfigManager,
+        source_filters: List[str],
+        target_filters: List[str],
+    ) -> Dict:
+        """Return dict objects formatted for yaml validations. Qualify the configurations by adding the source and
```

Comment:
Are these the filters that are provided with `data-validation generate-table-partitions -filters 'X'`? Or are these the new generated filters for the table partitions?